src/classes/ExtendedGraph.class.php

```diff
@@ -1643,7 +1783,7 @@ function get_label_for_uri($uri)
         }
         if (!isset(self::$labelProperties))
         {
-            throw new TripodException('Please initialise ExtendedGraph::$labelProperties');
+            throw new \Tripod\Exceptions\Exception('Please initialise ExtendedGraph::$labelProperties');
```

Comment:
Is \Tripod\Exceptions\Exception the right name here? If there is just one type of tripod exception, should it be up one level: \Tripod\Exception. If there are multiple exceptions for different reasons, then I guess there own package/directory/level makes sense - but then Exception would be a base exception?